ui/components/multichain/menu-items/view-explorer-menu-item.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { BtcAccountType } from '@metamask/keyring-api';
+import { fireEvent, renderWithProvider } from '../../../../test/jest';
+import configureStore from '../../../store/store';
+import mockState from '../../../../test/data/mock-state.json';
+import { createMockInternalAccount } from '../../../../test/jest/mocks';
+import {
+  MULTICHAIN_NETWORK_TO_EXPLORER_URL,
+  MultichainNetworks,
+} from '../../../../shared/constants/multichain/networks';
+import { ViewExplorerMenuItem } from '.';
+
+const mockAccount = createMockInternalAccount({
+  name: 'Account 1',
+  address: '0x0dcd5d886577d5081b0c52e242ef29e70be3e7bc',
+});
+
+const mockNonEvmAccount = createMockInternalAccount({
+  address: 'bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq',
+  // @ts-expect-error no type in createMockInternalAccount
+  type: BtcAccountType.P2wpkh,
```

Comment:
We do have one? https://github.com/MetaMask/metamask-extension/blob/develop/test/jest/mocks.js#L176

Maybe there's a typing issue though... We should probably allow `EthAccountType | BtcAccountType` for this field!